djangoevents/__init__.py

```diff
@@ -19,9 +21,20 @@
     'entity_mutator',
     'singledispatch',
     'publish',
+    'store_event'
     'subscribe',
     'unsubscribe',
     'subscribe_to',
     'BaseEntity',
     'EventSourcingWithDjango'
 ]
+
+
+def publish(event):
+    warnings.warn("`publish` is depreciated. Please switch to: `store_event`", DeprecationWarning)
+    return es_publish(event)
+
+
+def store_event(event):
```

Comment:
We want to discourage usage of subscribe because it doesn't make sense in most situations in Django application context (apart of storing the event, that is). This library exists mainly to simplify way of storing events in a database. For pub/sub there are far better options - like celery or huey. Remember that Django apps operates within web context most of the time - it doesn't really make sense to subscribe to those events in synchronous way as it could potentially block web workers. Again, maybe it would be better to discuss this offline. In my view this library should provide easy way of defining events and storing them using Django's ORM. For anything else there are better tools.